aiven/client/cli.py

```diff
@@ -2040,9 +2078,28 @@ def service__topic_create(self):
     @arg.min_insync_replicas
     @arg.retention
     @arg.retention_bytes
+    @arg.tagupdate
+    @arg.untag
     @arg("--replication", help="Replication factor", type=int, required=False)
     def service__topic_update(self):
         """Update a Kafka topic"""
+
+        new_tags = list(map(parse_tag_str, self.args.topic_option_tag))
+        untags = list(map(parse_untag_str, self.args.topic_option_untag))
+        tag_keys = list(map(lambda d: d.get("key"), new_tags)) + untags
+        if len(set(tag_keys)) != len(tag_keys):
+            for k in set(tag_keys):
+                tag_keys.remove(k)
+            raise argx.UserError(f"Duplicate tags detected: {', '.join(tag_keys)}")
+
+        tags = None
+        if len(new_tags) + len(untags) > 0:
+            topic = self.client.get_service_topic(
```

Comment:
This needs a comment. Is it necessary to fetch the existing tags because `update_service_topic` overwrite all the tags instead of adding on top?